docs/order.rb

```diff
@@ -0,0 +1,35 @@
+require './docs/menu' 
+require './docs/SMS'
+require 'rubygems'
+
+class Order
+
+  attr_reader :current_order
+
+  def initialize
+    @current_order = []
+    @new_menu = Menu.new
+    @order_price = 0
+  end
+
+  def add_to_basket(dish, quantity)
+    raise "Item not on the menu" unless @new_menu.menu.has_key?(dish)
+    quantity.times { @current_order << dish }
+  end
+
+  def calculate_price
```

Comment:
Think some of these methods could be encapsulated into private as the the user doesn't need to calculate the price, the program just needs to do it and the user sees the output